### DISCUSSION TITLE
原地转圈动作需要进行闭环

### DISCUSSION THREAD

**Discussion by geekroscom** [MiRoboticsLab/cyberdog_ros2#13] [General]
目前通过Python调用proto的MONO_ORDER_TURN_AROUND可以实现原地转圈，但是转圈的角度木有闭环，导致无法精准控制，希望尽快迭代，增加相关的闭环。

**Comment by geekroscom** [comment]
是否可以改为0--360度旋转角度控制？

**Reply by homalozoa** [reply]
可以加入任意角度控制

**Comment by geekroscom** [comment]
也可以增加对旋转圈数的控制？
